### PR TITLE
Modify rule S6332: Update issue message

### PR DESCRIPTION
**Pull request by pierre-loup-tristant-sonarsource**
Ticket: [SONARIAC-403](https://jira.sonarsource.com/browse/SONARIAC-403)

### CODE REVIEW

File: rules/S6332/cloudformation/message.adoc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+=== Message 
+
+* Make sure that using unencrypted EFS file systems is safe here.
+* Omitting "Encrypted" disables EFS file systems encryption. Make sure it is safe here.
```
Comment (loris-s-sonarsource) [inline rules/S6332/cloudformation/message.adoc:4]:
```suggestion
* Omitting the "Encrypted" attribute disables EFS file system encryption. Make sure it is safe here.
```

File: rules/S6332/terraform/message.adoc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+=== Message 
+
+* Make sure that using unencrypted EFS file systems is safe here.
+* Omitting "encrypted" disables EFS file systems encryption. Make sure it is safe here.
```
Comment (loris-s-sonarsource) [inline rules/S6332/terraform/message.adoc:4]:
```suggestion
* Omitting the "encrypted" attribute disables EFS file system encryption. Make sure it is safe here.
```

File: rules/S6332/cloudformation/message.adoc (hunk 1 above)
Comment (pierre-loup-tristant-sonarsource) [reply rules/S6332/cloudformation/message.adoc]: I was referring to the fact that all other issue messages share the pattern `Omitting "{parameter}" ...`, I thought it was better for this PR to stick to the existing pattern.
In general I am ok switching to another pattern like the one you suggested `Omitting the "{parameter}" attribute ...`

Comment (loris-s-sonarsource) [reply rules/S6332/cloudformation/message.adoc]: Ah ok, I see! Yes, we should do it in a separate future PR

**Pull request merged**